hivemind-host/proto/message.proto

```diff
@@ -1,9 +1,63 @@
 syntax = "proto3";
-import "generic.proto";
-import "user-call.proto";
 
 option java_package = "com.swarmus.hivear";
 
+// Generic
+enum GenericResponseStatus {
+    OK = 0;
+    BAD_REQUEST = 1;
+    ERROR = 2;
+}
+
+message GenericResponse {
+    GenericResponseStatus status = 1;
+    string details = 2;
+}
+
+// FunctionCall
+message FunctionArgument {
```

Comment:
is it possible to add an optional argument name here?
Would be nice to retrieve what is the argument about

Reply:
It's out of scope of what I'm doing as of now. And maybe it will be a whole other request to get argument types and names since the FunctionArgument are use to call a function for now, not get the  parameters type/name. But don't worry it's in the plans, you will have access to the argument names/types eventually to show on the UI.